src/components/datagrid/DataGrid.tsx

```diff
@@ -74,7 +74,7 @@ export const DataGrid: DatagridComponent = ({
             .reduce((accumulatedChildWidth, currentChildWidth) => accumulatedChildWidth + currentChildWidth, 0),
         )
         .reduce((maxRowWidth, currentRowWidth) => (currentRowWidth > maxRowWidth ? currentRowWidth : maxRowWidth), 0),
-    [rowNodes, rowNodes.size],
+    [rowNodes],
```

Comment:
I think this useMemo will now run only once, as `rowNodes` never seems to be changed and is always the same Map. That's whay the `rowNodes.size` was added I guess.